.github/workflows/cron-checks.yml

```diff
@@ -47,8 +45,6 @@ jobs:
     - name: Allure TestOps Launch Removal
       if: cancelled()
       run: bundle exec fastlane allure_launch_removal launch_id:$LAUNCH_ID
-      env:
-        ALLURE_TOKEN: ${{ secrets.ALLURE_TOKEN }}
```

Comment:
Why are we removing this? 🤔 How allure will work without the allure's token?

Reply:
It was propagated in two places in the pipeline: on the job level and on the step level. Just removed it from the step level, as it is used in multiple steps.